README.md

```diff
@@ -1,4 +1,4 @@
-# Borg for Yunohost
+# Borg Backup for Yunohost
```

Comment:
Btw today a user reported being ~scared by the fact that the app is still said  to be experimental a few lines below :s I dunno if it should be deemed "not experimental" now or not

Reply:
Commit by zamentur